git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -59,6 +61,10 @@ Before you start this module, make sure you
 
 Open a new command line interface on your computer. All exercises in this module will be done in this interface.
 
+<div class = "warning">
+If you are on a Windows machine, make sure you are in the Git Bash application. The Command Prompt (cmd) will not recognize the commands we are using in this module.  
+</div>
+
 If you are not sure if Git is configured on your computer, enter `git config --list` into your interface. The last two lines of output should show your `user.name` and `user.email`. If these are correct you are all ready to start the module!
 
 If you need to get Git configured on your computer the [Setting Up Git](https://liascript.io/course/?https://raw.githubusercontent.com/arcus/education_modules/rmh-git-setup/git_setup/setting_up_git.md#1) module will help you get set up in about 20 minutes.
```

Comment:
```suggestion
If you need to get Git configured on your computer the [Setting Up Git](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/setting_up_git_mac_and_linux/setting_up_git_mac_and_linux.md#1) module will help you get set up in about 20 minutes if you're using a Mac or Linux machine.
```

Not sure about how to word this while Windows is still left out of the fun, maybe stage this for integration after the Win setup?

Reply:
Definitely think we should get the Windows installation integrated and then just link to it after this.